src/components/Checkbox/Checkbox.js

```diff
@@ -16,51 +19,67 @@ const Checkbox = ({
   isMyAccount,
   className,
   disabled,
-  isRadioButton
+  isRadioButton,
+  termsUrl,
+  isPayPal
 }) => {
   const [isChecked, setIsChecked] = useState(checked);
+  const { t } = useTranslation();
 
   useEffect(() => {
     setIsChecked(checked);
   }, [checked]);
 
   return (
-    <CheckboxStyled
-      onClick={e => {
-        e.stopPropagation();
-        onClickFn(e, disabled, setIsChecked);
-      }}
-      role="checkbox"
-      tabIndex="-1"
-      aria-checked={isChecked}
-      checked={isChecked}
-      aria-label={children}
-      className={className}
-      disabled={disabled}
-    >
-      <CheckFrameStyled
-        error={error && required && !isChecked}
-        tabIndex="0"
-        onKeyDown={e => (e.keyCode === 32 ? onClickFn() : null)}
-        isMyAccount={isMyAccount}
-        isRadioButton={isRadioButton}
-        checked={isChecked}
-      >
-        {isChecked && (
-          <CheckMarkStyled
-            data-testid="checkmark"
-            isMyAccount={isMyAccount}
-            isRadioButton={isRadioButton}
-          />
-        )}
-      </CheckFrameStyled>
-      <ConsentDefinitionStyled
-        dangerouslySetInnerHTML={{
-          __html: `${children}${required && isMyAccount ? '*' : ''}`
+    <>
+      <CheckboxStyled
+        onClick={e => {
+          e.stopPropagation();
+          onClickFn(e, disabled, setIsChecked);
         }}
+        role="checkbox"
+        tabIndex="-1"
+        aria-checked={isChecked}
         checked={isChecked}
-      />
-    </CheckboxStyled>
+        aria-label={children}
+        className={className}
+        disabled={disabled}
+      >
+        <CheckFrameStyled
+          error={error && required && !isChecked}
+          tabIndex="0"
+          onKeyDown={e => (e.keyCode === 32 ? onClickFn() : null)}
+          isMyAccount={isMyAccount}
+          isRadioButton={isRadioButton}
+          checked={isChecked}
+        >
+          {isChecked && (
+            <CheckMarkStyled
+              data-testid="checkmark"
+              isMyAccount={isMyAccount}
+              isRadioButton={isRadioButton}
+            />
+          )}
+        </CheckFrameStyled>
+        <ConsentDefinitionStyled
+          dangerouslySetInnerHTML={{
+            __html: `${children}${required && isMyAccount ? '*' : ''}`
+          }}
+          checked={isChecked}
+        />
+      </CheckboxStyled>
+      {termsUrl && (
+        <TermsLinkStyled
+          href={termsUrl}
+          target="_blank"
+          rel="noreferrer"
+          checked={isChecked}
+          isPayPal={isPayPal}
```

Comment:
```suggestion
          $isPayPal={isPayPal}
```

We talked about this `transientProps` thing, so maybe we could use it here